### DISCUSSION TITLE
ReadHiddenChannels

### DISCUSSION THREAD

**Discussion by Velthyra** [JustOptimize/return-ShowHiddenChannels#84] [Q&A] [answered]
Is there a way to read hidden channels? Or do you have such plans? Or is this something completely impossible? I don't think it's impossible, but I know it requires a lot of coding knowledge and a lot of thought. If you ask why, as a result, you can show channel permissions and the date of the last message you can pull it to the panel. I would appreciate your answers, thank you. Thanks for plugin.

**Comment by JustOptimize** [comment, marked as answer]
As the readme says it's completely impossible to read the messages of hidden channels
